### DISCUSSION TITLE
Mozilla site rating now D+

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3532] [Development]
The site rating as reported in the [README](https://github.com/corona-warn-app/cwa-website/blob/master/README.md) has gone down from A+ to D+.

I don't know if this is important or not.

The current rating badge is here:

<a href="https://observatory.mozilla.org/analyze/coronawarn.app" title="Latest Results"><img src="https://img.shields.io/mozilla-observatory/grade/coronawarn.app" alt="Mozilla HTTP Observatory Grade"></a>

**Comment by Ein-Tim** [comment]
For reference, here is the history of how the site was rated in the past:

![Bildschirmfoto 2023-06-02 um 09 38 49](https://github.com/corona-warn-app/cwa-website/assets/67682506/e4c25a5b-f92f-4ffa-a9fc-6feefadf3d87)

**Reply by MikeMcC399** [reply]
If this badge is not important, it would be better to remove it instead of showing a bad rating.
